### ISSUE TITLE
Webpage does not render properly on IPADOS

### ISSUE THREAD

**Issue by Mgomez-01** [doublez13/ta-queue#64]
Webpage working fine on other devices but does not show up as intended when using an Ipad. Tested on Safari as well as Firefox browser for **IpadOS**. Screengrabs of the two browsers tested below. 

**Firefox**

![8E754737-2814-465C-BBAD-9BE14F94C06F](https://user-images.githubusercontent.com/70176273/152384372-b9522114-ff25-4dc7-ae07-c84c793b343c.png)

**Safari**

![35740B66-BFC9-4D4D-AEDB-550F755F212E](https://user-images.githubusercontent.com/70176273/152384454-0f5ec830-8c95-40b0-8543-cbc8ec4ec73e.png)

**Comment by doublez13** [comment]
Any chance you have JavaScript disabled?

Inside Firefox, are you able to look at the Console log? Should be under Web Developer Tools. Not sure if it's there on FF mobile though

**Comment by Mgomez-01** [comment]
Javascript is enabled under Safari settings, and Firefox does not seem to have an option to view the console on mobile

**Comment by doublez13** [comment]
I've made a small change on the dev site. Can you see if it's any different here: https://ta-dev.eng.utah.edu/

**Comment by Mgomez-01** [comment]
Yeah, it looks like the changes you made worked.

**Comment by doublez13** [comment]
K just double checking. Do a hard refresh of the page to load the changed javascript. It should be broken again

**Comment by Mgomez-01** [comment]
That is correct, it did break the page again. Sorry for leaving you hanging. I am also in the middle of a ECE3500 lecture

**Comment by doublez13** [comment]
Can you give it one more try now on the dev site (https://ta-dev.eng.utah.edu/). If that works, I'll push it do the stable version.

**Comment by Mgomez-01** [comment]
Yeah, it is definitely working. I see it as It shows up on my desktop. Thanks so much for taking care of it!
